src/components/Songslist/index.jsx

```diff
@@ -0,0 +1,24 @@
+import playersongs from '../../assets/imgs/button-play.jpg'
+import heart from '../../assets/imgs/Heart.png'
+import './style.css';
+
+function Songslist(){
```

Comment:
O nome em inglês não seia SongsList do componente?
```suggestion
function SongsList(){
```